test/performance/core/simd/simd_iota_benchmark.cpp

```diff
@@ -15,17 +15,17 @@
 #include <seqan3/core/simd/view_iota_simd.hpp>
 
 // ============================================================================
-//  simd_iota_iterator
+//  iota_simd_view_benchmark
 // ============================================================================
 
 template <typename simd_t>
-struct simd_iota_iterator_function
+struct iota_simd_view_benchmark
 {
     using scalar_type = typename seqan3::simd::simd_traits<simd_t>::scalar_type;
 
     seqan3::detail::iota_simd_view<simd_t> simd_iota_view;
```

Comment:
```suggestion
    seqan3::detail::iota_simd_view<simd_t> iota_simd_view;
```

Sooryyy :see_no_evil: